README.md

```diff
@@ -23,6 +23,11 @@ It includes a modified version of [cjlano's python SVG parser and drawing module
 
 ```pip install svg2mod```
 
+#### Do not want to install the application? 
```

Comment:
Could this be changed to a weight 3 `###`?

Reply:
I have changed it to heading level 3